### DISCUSSION TITLE
Adding `--destination` flag to 10.2.X

### DISCUSSION THREAD

**Discussion by radiodario** [react-native-community/cli#2074] [Q&A] [answered]
Hi community - thank you so so much for building such a cool set of tools. Our team is stuck on react native 0.71 due to some dependency issues that are forcing us to run rosetta sims and i was just wondering if it would be possible to bring in the changes from #1934 onto a new patch release of the CLI - or if any of you would know how to use a newer version of the CLI (i.e. 11) without upgrading to react native 0.72

Thanks!

**Comment by szymonrybczak** [comment, marked as answer]
Hey @radiodario, sorry for late response - creating patch would be probably long process, if you want to upgrade CLI without upgrading React Native version you can add relevant CLI packages (in your case probably `@react-native-community/cli-platform-ios`) to `package.json`, reinstall package and when running `npx react-native run-ios`, it should use the CLI from `package.json`. Please remember about this line, when upgrading in future because it can create some issues.

**Reply by radiodario** [reply]
Thanks a lot @szymonrybczak - useful to know, I tried installing the desired version manually but to no avail. We are in fact upgrading to 0.72 this week so hopefully this won't be an issue anymore

**Reply by szymonrybczak** [reply]
Alright, I hope the upgrade will be successful! 🤞 Let me know if you run into any issues.
